notebooks/community/ml_ops/stage6/get_started_with_custom_tabular_model_batch.ipynb

```diff
@@ -716,14 +716,19 @@
         "\n",
```

Comment:
<p>Can we add a link to learn more about training a custom model here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
In the preamble, under the subsection Model, I describe where the pretrained models came from